pages/guides/configuration/Creating-a-Pack.md

```diff
@@ -1,99 +1,216 @@
+    ```yaml
+    id: TUTORIAL
+    author: Astrash # Optional
+    version: 1.0.0  # Optional
+    ```
 
-    ###### EDITOR TIP
+You should now have a config file called `pack.yml` inside your pack directory, which contains a pack ID, and optionally an author and or version.
 
-    > We highly recommend using a text editor or IDE such as VSCode or InteliJ, which provides support for YAML syntax
-    > highlighting and file browsing support, as it will make things much more streamlined and easy for you, the config
-    > editor!
+---
 
-5. Set up an ID for your new pack:
+</details>
 
-    1. In your text editor, open up `pack.yml`, found in the root directory of the new config pack.
+<details>
 
-    2. Within `pack.yml`, Change the `id` to the ID you wish to assign your
-config, as well as the `author` to your username like so:
+<summary><b>3.</b> <i>After creating a pack manifest, you will need to <b>create a biome configuration</b> so the generator will have a biome to generate.</i></summary>
+
+---
+
+- Create a new folder within your pack directory called `biomes`.
+  
+- Within the new `biomes` folder, create a new config file with a name of your choice. This file is your *biome configuration*.
+
+    > The file name of a biome configuration doesn't matter (aside from the extension) but generally you'll want to name it after the name of your biome for organization - for this tutorial we will call the file `first_biome.yml`.
+
+- Add the following lines to your new biome configuration:
+
+    ```yaml
+    id: <Biome ID>
+    vanilla: <Vanilla Biome ID>
+    noise-equation: "-y + <Base Height>"
+    palette:
+      - "BLOCK:minecraft:<Block ID>": 255
+    ```
+
+- **Replace the placeholders** (the things surrounded by angle brackets `< >`) with the relevant information. You're free to use whatever parameters you'd like, but if you need some reference, here is an example with each placeholder filled in, as well as explanations of what each parameter does:
 
     ```yaml
-    id: EXAMPLE_ID
-    # Replace 'EXAMPLE_ID' with your desired pack ID
+    id: FIRST_BIOME
+        # The ID of your biome.
+        # (This is similar to the pack ID in regards to naming conventions)
+    
+    vanilla: PLAINS
+        # A vanilla biome ID - used for aspects like grass color, mob spawning, etc..
+        # You can find a list of valid biome IDs on the minecraft wiki:
+        # https://minecraft.fandom.com/wiki/Biome#Biome_IDs
+
+    noise-equation: "-y + 64"
+        # The mathematical equation that determines the shape of terrain.
+        # How exactly this works is a more advanced topic we will cover later on.
+        # For now, just know this equation will generate flat land at Y = 64.
+
+    palette:
+        # A list of blocks that the terrain in the biome will consist of.
+        # In this case, Terra will generate stone everywhere there is land below Y = 255.
+      - "BLOCK:minecraft:stone": 255
+    ```
```

Comment:
This example should probably go in a spoiler

Reply:
I'm gonna leave it as is since I think the example is still 'part of the step' rather than being something optional. New users should probably read it anyway so hiding it might make users skim over it